src/System.Diagnostics.TraceSource/src/System/Diagnostics/TraceSource.cs

```diff
@@ -437,7 +437,7 @@ public SourceSwitch Switch
             set
             {
                 if (value == null)
-                    throw new ArgumentNullException("Switch");
+                    throw new ArgumentNullException(nameof(Switch));
```

Comment:
Should probably be corrected to `nameof(value)` (remember, the `ArgumentNullException` takes a [parameter name](https://msdn.microsoft.com/en-us/library/wssa019h(v=vs.110).aspx), not a member one).